dart/dynamics/Shape.h

```diff
@@ -121,7 +126,7 @@ class Shape : public virtual common::Subject
   //           biased mesh shape. Two Vector3ds might be better; one is for
   //           minimum verterx, and the other is for maximum verterx of the
   //           bounding box.
-  const Eigen::Vector3d& getBoundingBoxDim() const;
+  Eigen::Vector3d getBoundingBoxDim() const;
```

Comment:
There is a `DEPRECATED` macro defined in [`Deprecated.h`](https://github.com/dartsim/dart/blob/e8b4f1c40e26140d93ea803bcb4a76547ebe05fb/dart/common/Deprecated.h) that has `#ifdef`s for all of the supported compilers. The argument is the major/minor version number when the deprecation occurred.

DART 5.1.1 was just released, so I suppose it should be `DEPRECATED(5.2)` now?